### DISCUSSION TITLE
Would you guys bring the update to the map that have a dashed border between Kosovo and Serbia in Balkan?

### DISCUSSION THREAD

**Discussion by FarukNetworks** [Leaflet/Leaflet#9497] [Q&A] [answered]
Would you guys bring the update to the map that have a dashed border between Kosovo and Serbia in Balkan? because I'm working with the Balkan company about some legal staff and they are very sensitive about these staff, so it's basically the true map of the Balkan countries. You can check off the google maps. 

![image](https://github.com/user-attachments/assets/e1fd2341-7216-49ad-887e-22bd949a8445)

![image](https://github.com/user-attachments/assets/444aede5-3cf8-4aca-a31d-4602ffd4c571)

**Comment by jonkoops** [comment, marked as answer]
We don't provide the map data, so no we can't.

**Reply by IvanSanchez** [reply]
This is brought up every once in a while - see also https://github.com/Leaflet/Leaflet/issues/5056
